x-pack/solutions/observability/plugins/observability_shared/public/hooks/use_url_state.ts

```diff
@@ -77,7 +75,7 @@ export const useUrlState = <State>({
         history.replace(newLocation);
       }
     },
-    [decodeUrlState, defaultState, encodeUrlState, history, urlStateKey]
```

Comment:
`defaultState` was causing `setState` function to be redefined during re-renders